src/store/coinListStore/requestCoinList.ts

```diff
@@ -0,0 +1,39 @@
+import { Options } from "@components/UI/Dropdown";
+import { apiUrls } from "@config/apiUrls";
+import { CoinListModel, normalizeCurrency } from "@store/models";
+import { normalizeCoinListApiModel } from "@store/models";
+import { ApiResp } from "@utils/apiTypes";
+import axios from "axios";
+export const requestCoinList = async (
```

Comment:
пустая строка после импортов
```suggestion
import axios from "axios";

export const requestCoinList = async (
```